packages/aws-cdk-lib/aws-elasticloadbalancingv2/lib/alb/application-listener.ts

```diff
@@ -188,12 +245,32 @@ export class ApplicationListener extends BaseListener implements IApplicationLis
       throw new Error('At least one of \'port\' or \'protocol\' is required');
     }
 
+    if (props.mutualAuthentication?.mode === Mode.VERIFY
+      && !Token.isUnresolved(props.mutualAuthentication.trustStore) && !props.mutualAuthentication.trustStore) {
+      throw new Error('You must set \'trustStore\' when \'mode\' is \'Mode.VERIFY\'');
+    }
```

Comment:
```suggestion
    if (props.mutualAuthentication?.mode === Mode.VERIFY && !props.mutualAuthentication.trustStore) {
      throw new Error(`You must set 'trustStore' when 'mode' is '${Mode.VERIFY}'`);
    }
```
I expect `trustStore` to always be resolved.